[2022-07-06 15:42:15,494] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-07-06T15:41:51.506436+00:00 [queued]>
[2022-07-06 15:42:15,528] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-07-06T15:41:51.506436+00:00 [queued]>
[2022-07-06 15:42:15,529] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-06 15:42:15,534] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-06 15:42:15,535] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-06 15:42:15,609] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): accurate> on 2022-07-06 15:41:51.506436+00:00
[2022-07-06 15:42:15,630] {standard_task_runner.py:52} INFO - Started process 267 to run task
[2022-07-06 15:42:15,670] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'accurate', 'manual__2022-07-06T15:41:51.506436+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/my_ml_dag.py', '--cfg-path', '/tmp/tmpayc8dibu', '--error-file', '/tmp/tmpidwy45is']
[2022-07-06 15:42:15,681] {standard_task_runner.py:80} INFO - Job 29: Subtask accurate
[2022-07-06 15:42:16,239] {task_command.py:370} INFO - Running <TaskInstance: my_dag.accurate manual__2022-07-06T15:41:51.506436+00:00 [running]> on host fce85b9f2669
[2022-07-06 15:42:16,792] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-07-06T15:41:51.506436+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-06T15:41:51.506436+00:00
[2022-07-06 15:42:16,801] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-06 15:42:16,806] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-07-06 15:42:16,855] {subprocess.py:85} INFO - Output:
[2022-07-06 15:42:16,868] {subprocess.py:92} INFO - accurate
[2022-07-06 15:42:16,873] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-06 15:42:17,167] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=accurate, execution_date=20220706T154151, start_date=20220706T154215, end_date=20220706T154217
[2022-07-06 15:42:17,399] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-06 15:42:17,804] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
